src/frontend/App.tsx

```diff
@@ -48,7 +48,13 @@ const mapeoApi = createMapeoClient(messagePort, {timeout: Infinity});
 const localDiscoveryController = createLocalDiscoveryController(mapeoApi);
 localDiscoveryController.start();
 initializeNodejs();
-SplashScreen.preventAutoHideAsync();
+
+SplashScreen.setOptions({
+  fade: true,
+});
+SplashScreen.preventAutoHideAsync().catch(err => {
+  console.log(err);
+});
```

Comment:
as mentioned, these won't seem to work when you're working with a dev build on newer Android versions